VideoWeb/VideoWeb.AcceptanceTests/Steps/VhoVenueListSteps.cs

```diff
@@ -39,10 +39,22 @@ public void SelectVenues(string venues)
             {
                 _browsers[_c.CurrentUser.Key].Driver.FindElement(VhoVenueAllocationPage.VenuesTextBox).SendKeys(venue);
                 _browsers[_c.CurrentUser.Key].ClickCheckbox(VhoVenueAllocationPage.VenueCheckbox(venue));
-                DeleteText(venue);
             }
         }
-        
+
+        [When(@"the VHO selects the hearings for Judge named (.*)")]
+        public void WhenTheVHOSelectsTheHearingsForJudgeNamedAutomationCourtroom(string judgeNames)
+        {
+            _browsers[_c.CurrentUser.Key].Driver.WaitUntilVisible(VhoVenueAllocationPage.VenuesDropdown).Displayed.Should().BeTrue();
+            _browsers[_c.CurrentUser.Key].Click(VhoVenueAllocationPage.VenuesTextBox);
+
+            foreach (var venue in ConverterHelpers.ConvertStringIntoArray(judgeNames))
+            {
+                _browsers[_c.CurrentUser.Key].Driver.FindElement(VhoVenueAllocationPage.VenuesTextBox).SendKeys(venue);
```

Comment:
For checkboxes it should be Driver.WaitUntilExists

Reply:
this is for the drop down.

Reply:
_browsers[_c.CurrentUser.Key].Driver.FindElement(VhoVenueAllocationPage.VenuesTextBox).SendKeys(venue);

Reply:
okay it should be WaitUntilVisible then. We don't use FindElement because it won't wait if the page takes time to load

Reply:
updated to WaitUntilVisible